core/src/main/scala/scalaz/StrictTree.scala

```diff
@@ -11,7 +11,7 @@ import std.vector.{vectorInstance, vectorMonoid}
   */
 case class StrictTree[A](
   rootLabel: A,
-  subForest: Vector[StrictTree[A]]
+  subForest: List[StrictTree[A]]
```

Comment:
I find the `List` tree to perform better than the `Vector` tree for all operations excepting those for which `Vector` has decidedly better characteristics, the most notable of those being `foldRight`.